hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterServiceImpl.java

```diff
@@ -314,6 +315,23 @@ public void reset() {
         }
     }
 
+    private void convertLocalMemberToLite() {
+        previouslyDataMember = !localMember.isLiteMember();
+        localMember.setLiteMember(true);
+
+        if (previouslyDataMember) {
+            logger.info("Converted local data-member to lite-member before start of split brain healing");
+        }
+    }
+
+    void tryPromoteLocalLiteMember() {
```

Comment:
I think we can remove this method and directly call `promoteLocalLiteMember()`

Reply:
please see cluster-merge-task